src/ExpressionParser.php

```diff
@@ -53,7 +53,7 @@ class ExpressionParser
     private array $unaryOperators;
     /** @var array<string, array{precedence: int, class: class-string<AbstractBinary>, associativity: self::OPERATOR_*}> */
     private array $binaryOperators;
-    private $readyNodes = [];
+    private array $readyNodes = [];
```

Comment:
Removed in 2acd9e8d9fcf2fa7907db8a6e1bcdd8feffcd91f

Reply:
Thanks, rebased and removed